python/lsst/analysis/tools/interfaces/datastore/_sasquatchDatastore.py

```diff
@@ -80,6 +81,9 @@ class SasquatchDatastore(GenericBaseDatastore):
     namespace: str
     """The namespace in Sasquatch where the uploaded metrics will be
     dispatched.
+
+    Defaults to the value obtained from the ``$DAF_BUTLER_SASQUATCH_NAMESPACE``
+    environment variable and "lsst.dm" if no environment variable is defined.
```

Comment:
The wording here is a bit confusing, assuming I understand correctly that these fields can _all_ be set directly in config. Maybe:
```suggestion
    Overridden by the ``$DAF_BUTLER_SASQUATCH_NAMESPACE`` environment variable
    if it is defined. Defaults to "lsst.dm".
```

Reply:
Should this have an explicit default like `defaultConfigFile` does?

Reply:
Sorry. I don't know what you mean. The default is `lsst.dm`.

Reply:
`__init__` assigns "lsst.dm" if nothing supersedes that. I meant "should something be assigned to `namespace` here, where it's declared?"

Reply:
Ah. Okay. We could do that. Generally the class properties are there for docstrings and I'm not sure why Nate didn't set the default there instead, although a class property and an instance property can get confusing.